simple_history/models.py

```diff
@@ -311,3 +311,46 @@ def __get__(self, instance, owner):
         values = (getattr(instance, f.attname)
                   for f in self.model._meta.fields)
         return self.model(*values)
+
+
+class HistoricalChanges(models.Model):
+
+    class Meta:
+        abstract = True
+
+    def __sub__(self, old_history):
```

Comment:
My C++ background left me with a bad taste in my mouth for operator overloads. I tend to favor explicit method calls.
```python
delta = current.changesSince(prior)
```